airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -98,6 +98,15 @@ def __init__(
         log_id_template: str | None = None,
     ):
         es_kwargs = es_kwargs or {}
+        # For elasticsearch>8,arguments like retry_timeout have changed for elasticsearch to retry_on_timeout
+        # in Elasticsearch() compared to previous versions.
+        # Read more at: https://elasticsearch-py.readthedocs.io/en/v8.8.2/api.html#module-elasticsearch
+        if es_kwargs:
+            retry_timeout = es_kwargs.get("retry_timeout")
+            if retry_timeout:
+                es_kwargs["retry_on_timeout"] = retry_timeout
+                del es_kwargs["retry_timeout"]
```

Comment:
@sunank200 I think we need to be backward compatible with elasticsearch=7 as well right?

Reply:
yes @utkarsharma2 I am pushing the change now